tools/PI/DevHome.PI/Helpers/DesktopBridgeHelper.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+
+namespace DevHome.PI.Helpers;
+
+public class DesktopBridgeHelper

```

Comment:
These are good candidates for RuntimeHelper, e.g. IsMSIX is already implemented there. (Pretty sure the Win7 or lower check isn't valid for this app anyway :))

Reply:
Good point.